### PR TITLE
Make ImagePlaceholder available via the global wp object

### PR DESCRIPTION
**Pull request by johnwatkins0**
## Description

I noticed the `ImagePlaceHolder` component was missing from the global wp object. I assume this is not a deliberate choice, as `ImagePlaceHolder` is super useful. This PR fixes the issue by importing and exporting it in blocks/index.js.

### CODE REVIEW

File: blocks/index.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ export { default as BlockEdit } from './block-edit';
 export { default as BlockIcon } from './block-icon';
 export { default as ColorPalette } from './color-palette';
 export { default as Editable } from './rich-text/editable';
+export { default as ImagePlaceHolder } from './image-placeholder';
```
Comment (aduth) [inline blocks/index.js]: Given the naming of the directory, I'd expect this to be camel-cased as: `ImagePlaceholder`

**Review by aduth — CHANGES_REQUESTED** [review]

File: blocks/index.js (hunk 1 above)
Comment (johnwatkins0) [reply blocks/index.js]: Agreed, @aduth. The most recent commit changes `ImagePlaceHolder` to `ImagePlaceholder` (everywhere).

**Review by aduth — APPROVED** [review]
Looks great 👍

**Comment by mtias** [comment]
Thanks!

**Pull request merged**